test/unit/utils/validation.test.js

```diff
@@ -267,6 +270,32 @@ describe('validation', () => {
       });
     });
 
+    describe('Specific date format: FR', () => {
+      before((done) => {
+        i18n.off('languageChanged');
```

Comment:
I had to remove the listener on event "languageChanged" otherwise the method changeLanguage would not work...